pkg/controller/kogitoinfra/infinispan/infinispan_properties.go

```diff
@@ -109,5 +113,7 @@ func getInfinispanAppProps(cli *client.Client, name string, namespace string) (m
 		appProps[propertiesInfinispanSpring[appPropInfinispanServerList]] = infinispanURI
 		appProps[propertiesInfinispanQuarkus[appPropInfinispanServerList]] = infinispanURI
 	}
+	appProps[propertiesInfinispanSpring[appPropInfinispanSaslMechanism]] = saslPlain
+	appProps[propertiesInfinispanQuarkus[appPropInfinispanSaslMechanism]] = saslPlain
 	return appProps, nil
```

Comment:
So right now we always add all properties (Spring Boot and Quarkus) to the config map, no matter which runtime is running?

Reply:
yes, because the KogitoInfra can be reused by multiple services, both Quarkus and SpringBoot